src/components/form/wizard/footer.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import styled from 'styled-components'
+import Button from '../button'
+import { StepContext } from '.'
+import { localize } from 'components/localization'
+
+const StyledFooter = styled.div`
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    padding: 0 24px;
+    gap: 8px;
+    position: absolute;
+    height: 72px;
+    right: 0;
+    bottom: 0;
+    border-top: 1px solid var(--color-grey-43);
+    width: 100%;
+`
+
+const Footer = ({ disabled }: { disabled: boolean }) => {
+    const { step, setStep, max_step } = React.useContext(StepContext)
+
+    return (
+        <StyledFooter>
+            <Button
+                tertiary="true"
+                disabled={step <= 1}
+                onClick={() => step > 1 && setStep(step - 1)}
+            >
+                {localize('Previous')}
+            </Button>
+            <Button
+                secondary="true"
+                disabled={disabled}
+                onClick={() => step < 5 && setStep(step + 1)}
```

Comment:
can we create a function for `onClick`s, this function can have "previous" and "next" as enum or something like that?